package.json

```diff
@@ -15,7 +15,7 @@
         "build:ts": "tsc --build --verbose tsconfig.all.json",
         "build:other": "pnpm --recursive --workspace-concurrency=0 run build",
         "package": "pnpm --recursive --workspace-concurrency=0 run package",
-        "release": "pnpm run clean && pnpm run build:ts && pnpm run package && pnpm run build:other",
+        "release": "pnpm run clean && pnpm run build:ts && pnpm run package && pnpm run build:other && changeset publish && git push --follow-tags && git status",
```

Comment:
I wonder if we should add another command here, "release" was supposed to mean "(build for) release"

Reply:
Isn't `build` just build for release?

Reply:
`build` doesn't do a fresh build or shim the package.json files. I think we can just rename the current command to `build:release`, and then `release` can call it and do the other stuff

Reply:
I'll do this, although IMO building for release should really just be `build`. If there's additional work to be done before publishing it really should be in `prepare` or `prepublishOnly`, but we can sort that out later.